### DISCUSSION TITLE
Create Postal Email Accounts  and Connecting  to Mautic

### DISCUSSION THREAD

**Discussion by ripon202310** [postalserver/postal#3121] [Help with using Postal] [answered]
Hi everyone,

I'm new to both GitHub and Postal, so I apologize if my question seems basic. I haven't been able to find an answer in my searches, so I'm hoping you can help.

I've set up Postal on my OVH VPS and installed Mautic through Cyber Panel. My question is, how can I Create an email accounts from Postal (as like ripon@mydomain.com) and connect it to Mautic?

Thank you for your time and support!

Sincerely,
Ashiqur Rahman

**Comment by willpower232** [comment, marked as answer]
Postal doesn't have email accounts, it just sends emails. If you add your domain to Postal, presumably you create an SMTP credential and set that in Postal.

**Reply by ripon202310** [reply]
Thank you.
